components/common/header.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import styled from "styled-components";
+
+const HeaderStyle = styled.h1`
+  font-family: "Roboto Mono", monospace;
```

Comment:
You can delete `font-family` here because it is already being declared in the `global.css`.